syft/pkg/cataloger/sbom/cataloger.go

```diff
@@ -0,0 +1,43 @@
+package sbom
+
+import (
+    "bytes"
+    "fmt"
+    "github.com/anchore/syft/internal/formats/cyclonedxjson"
+    "github.com/anchore/syft/syft/artifact"
+    "github.com/anchore/syft/syft/pkg"
+    "github.com/anchore/syft/syft/pkg/cataloger/common"
+    "io"
+)
+
+// NewSBOMCataloger returns a new SBOM cataloger object loaded from saved SBOM JSON.
+func NewSBOMCataloger() *common.GenericCataloger {
+    globParsers := map[string]common.ParserFn{
+        "**/deps.json":         parseSBOM,
```

Comment:
I cannot find this pattern in the recognized file patterns for cyclonedx. (See https://cyclonedx.org/specification/overview/#recognized-file-patterns)

Where does this come from?

Reply:
I was just not aware of CycloneDX recognized file patterns. My bad. Will fix.

Reply:
Thanks for the fix! Functionally it looks great. We still have a few other open questions about functional and non-functional requirements around this feature that we will probably need more input from @wagoodman / @luhring and team.